ntbs-service/Pages/Notifications/Denotify.cshtml.cs

```diff
@@ -83,4 +79,4 @@ public async Task<IActionResult> OnPostConfirmAsync()
             return RedirectToPage("/Notifications/Overview", new { id = NotificationId });
```

Comment:
+Christian comment: Absolute path rather than relative as is the case everywhere else?

Reply:
@jasiekmiko Have you a preference on this? I would personally prefer absolute paths. Happy to relativ-ise for consistency though.